tests/test_setup_door.py

```
@@ -0,0 +1,372 @@
+        result = run_door("--new", str(target), env=env)
+        assert result.returncode == 0, result.stderr + result.stdout
+        assert "Project Created Successfully" in result.stdout  # export engine
+        assert "Install record committed." in result.stdout
+        _assert_doctor_tail(result.stdout)
+        region = _kit_install_region(target)
+        assert "shape: single" in region
+        assert "profile: python" in region
+        assert (target / "scripts" / "local" / "checks.sh").is_file()
+        assert (target / "scripts" / "local" / "kit_markers.py").is_file()
+        # export commit + record commit, nothing dangling
+        log = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)